MySQL-Core-Tests-Integration/MySQLServerStatementFetchTest.class.st

```diff
@@ -7,6 +7,24 @@ Class {
 	#category : #'MySQL-Core-Tests-Integration-Base'
 }
 
+{ #category : #tests }
+MySQLServerStatementFetchTest >> _testReadNextRow [
```

Comment:
this kind of name is not at all how conventions are (the underscore).
can you suggest a better name?

Reply:
Thanks for the comment; have rewritten the test cases longhand as that seems to be the general pattern elsewhere.